index.html

```diff
@@ -294,33 +294,33 @@ <h2 class="contact-h2">
                 </h2>
             </div>
             <form action="https://formspree.io/f/xnqrnjgr" class="contact-details" method="post">
-                <ul class="contact-list">
-                    <li>
-                        <input type="text" class="contact-1" maxlength="30" placeholder="Full name" required>
-                    </li>
-                    <li>
-                        <input type="email" class="contact-2" placeholder="Email address" required>
+                <ul class="contact-text"> 
+                    <li> 
+                        <input type="text" class="contact-10" maxlength="30" placeholder="Full name" required> 
                     </li>
+                    <li> 
+                        <input type="email" class="contact-20" placeholder="Email address" required>
+                    </li> 
                     <li>
-                        <textarea class="contact-3" maxlength="500" placeholder="Enter text here" required></textarea>
-                    </li>
+                        <textarea class="contact-30" maxlength="500" placeholder="Enter text here" required></textarea>
```

Comment:
- [ ] Please set the `name` attribute for your full name input tag so that Formspree will be able to collect data and save it and work as expected. An example is `<input name="message" id="message" type="text">` Please fix it :+1: